CONTRIBUTING.md

```diff
@@ -89,7 +89,7 @@ Manually testing Revel integration
 1. `cd` into the `examples/revelapp/` directory
 1. Run the `start-test-servers.sh` script
 1. Run `revel run`
-1. Visit `localhost:4321/handled` and verify that `payloads.json` now contains a report payload containing a handled payload, including a `session` sub-object containing 0 unhandled and 1 handled.
-1. Visit `localhost:4321/unhandled` and verify that `payloads.json` now contains an unhandled payload, including a `session` sub-object containing 1 unhandled and 0 handled.
+1. Visit `localhost:9001/handled` and verify that `payloads.json` now contains a report payload containing a handled payload, including a `session` sub-object containing 0 unhandled and 1 handled.
+1. Visit `localhost:9001/unhandled` and verify that `payloads.json` now contains an unhandled payload, including a `session` sub-object containing 1 unhandled and 0 handled.
 1. Let the application run for at least one minute and verify that sessions are being populated under the "sessions" array in `payloads.json`
```

Comment:
`payloads.json` did not contain a session payload after waiting for >1 minute, the following message was printed to the console:

> bugsnag/session.publish expected 202 response status, got HTTP 201 Created

Reply:
The response happens because 201 is the status code the simple `json-server` assumes it should return, but the session server will actually return 202.